README.md

```diff
@@ -189,6 +187,8 @@ whom they elect at their own discretion.
 
 ### TSC Emeriti
 
+* [joshgav](https://github.com/joshgav) -
+**Josh Gavant** &lt;josh.gavant@gmail.com&gt;
```

Comment:
nit: This list is sorted alphabetically by GitHub handle

Reply:
fixed!